tasks/add-relayer.ts

```diff
@@ -12,10 +13,14 @@ task("add-relayer", "Add relayer with given address")
     const bridgeAddress = config["bridge"];
     const bridgeContractFactory = await hre.ethers.getContractFactory("Bridge");
     const bridge = bridgeContractFactory.attach(bridgeAddress).connect(adminWallet);
-    await bridge.addRelayer(address);
+    if (taskArgs.price) {
```

Comment:
For every condition like this (see other files below), it would be better to have some getOptions function (or something like that). Because it would be easier to read the task logic without this if/else condition here.